lib/charms/saml_integrator/v0/saml.py

```diff
@@ -290,6 +290,18 @@ def _on_relation_changed(self, event: ops.RelationChangedEvent) -> None:
         if event.relation.data[event.relation.app]:
             self.on.saml_data_available.emit(event.relation, app=event.app, unit=event.unit)
 
+    def get_relation_data(self) -> typing.Optional[SamlRelationData]:
```

Comment:
why the optional? I understand that with the assert the intention is that it is not optional